toucan_data_sdk/utils/postprocess/sort.py

```diff
@@ -11,10 +11,15 @@ def sort(df, columns: List[str], order='asc'):
     ### Parameters
 
     *mandatory :*
-    - `columns` (*list*): list of the names of the columns to sort
+    - `columns` (*str* or *list(str)*): list of columns to order
 
     *optional :*
-    - `order` (*str*): 'asc' (default) or 'desc'
+    - `order` (*str* or *list(str)*): the ordering condition ('asc' for
+    ascending or 'desc' for descending). If not specified, 'asc' by default. If
```

Comment:
```suggestion
    ascending or 'desc' for descending). If not specified, 'asc' by default.
    If
```
Un ptit retour a la ligne x)